packages/vscode-extension/src/handlers.ts

```diff
@@ -835,6 +840,84 @@ export async function validateManifestHandler(args?: any[]): Promise<Result<null
     const workspacePath = globalVariables.workspaceUri?.fsPath;
     const manifestTemplatePath = `${workspacePath}/${AppPackageFolderName}/manifest.json`;
 
+    if (isValidationEnabled()) {
```

Comment:
Is this needed for CLI?

Reply:
Yes, CLI will be in another PR.

Reply:
Maybe include the questions in fx-core so no need to write it in both interfaces.